src/components/ListView/ListView.js

```diff
@@ -165,6 +165,12 @@ export class ListView extends Component {
     />
   );
 
+  rowUpdater = ({ id }) => {
+    const { checkboxList: { selectedRecords } } = this.props;
+
+    return selectedRecords && selectedRecords.has(id);
```

Comment:
I see that for similar cases below you have used just 
`return selectedRecords.has(id);`

Is there additional check is needed?

Reply:
The check is needed because `checkboxList` is not a required property. However, I am going to refactor it.

Reply:
Sounds good.